src/coffee/export.coffee

```diff
@@ -321,9 +321,9 @@ class Export
           Promise.reject err
 
   createTemplate: (languages, outputFile, allProductTypes = false) ->
-    @typesService.getAll(@client)
+    @typesService.getAll(@client, @projectKey)
     .then (result) =>
-      productTypes = result.body.results
+      productTypes = result
```

Comment:
Hi @wirtsi .. why not naming it `productTypes`  right away one line above?

Reply:
Right, I changed this ....